File: Selection_sort.cpp
#include<bits/stdc++.h>
using namespace std;

void selection_sort(vector<int>&arr){
    int n=arr.size();
    for(int i=0;i<n-1;i++){
        int smallest=i;
        for(int j=i+1;j<n;j++){
            if(arr[j]<arr[smallest]){
                swap(arr[j],arr[smallest]);
            }
        }
    }
}

int main(){
    vector<int>arr={1,2,3,12,35,4,6,78,98};
    selection_sort(arr);
    for(int i=0;i<arr.size();i++){
        cout<<arr[i]<<" ";
    }
}

File: Selection_sort.py
def selection_sort(arr):
    n=len(arr)
    for i in range (n-1):
        min_index=i
        for j in range(i+1,n):
            if arr[j]<arr[min_index]:
                min_index=j
        arr[i],arr[min_index]=arr[min_index],arr[i]

arr=[1,2,3,12,4,5,6,7,8,9,10]
print(f"Original List:{arr}")
selection_sort(arr)
print(f"Sorted List:{arr}")
